### ISSUE TITLE
Better color widget

### ISSUE THREAD

**Issue by pierreporte** [OtterBrowser/otter-browser#1282]
The current colour widget has some flaws and could be improved. Despite displaying the colour correctly, its text can be unreadable if the selected colour is the same as the colour of the text. For example, a very dark colour, nearly black, would make the widget unreadable with most themes. Also, the menu opened after clicking the widget gives the opportunity to copy the colour, however when it is empty (invalid), it returns #000000, which is black when you would expect nothing at all.

This is my proposition:

![s lection_559](https://cloud.githubusercontent.com/assets/2029853/23083282/3fb83948-f55d-11e6-96bf-4680ccb67197.png)

Basically, this new widget is composed of two simple widgets: a button and a label or a text box. The button is just a rectangle that displays the selected colour. It is possible to chose a new colour by clicking on it which display a color picker. The second widget displays the colour name that can be copied and edited like standard text. When no colour is selected the text could be “None” and the button could display an X or something similar. This way, the colour and its name are clearly separated and always readable, and edition is easier.
